src/views/TimelineTrackView.tsx

```diff
@@ -85,7 +85,9 @@ export default function TimelineTrackView(
             </div>
         </div>
         <div id={"gait-suggestion-div"}>
-            {/* TODO: add buttons */}
+            <button style={{left:"0%"}} />
+            <button style={{left:"calc(50% - 2px)"}} />
```

Comment:
with your button width as 10px, you'll want to subtract half the button width to center it on the desired timeline percentage

Reply:
this is fine for now though since these are only placeholders

Reply:
Yes, that I was thinking just to get a visual of where they would be.